### PR TITLE
added weight choice to first and next question

### PR DESCRIPTION
**Pull request by ZcortezZ**

### CODE REVIEW

File: edufinder/edufinder/rest_api/views.py [hunk 1]
```diff
@@ -68,8 +69,9 @@ def get_nextquestion(previous_answers: List[dict]):
     Expected input format
         [ { id: int, question: str, answer: int }, ... ]
     """
-    return random.choice(list(set([x.id for x in Question.objects.all()]) -
-                              set([x['id'] for x in previous_answers])))
+    used_ids = [x['id'] for x in previous_answers]
+    return Question.objects.exclude(pk__in=used_ids).order_by('-weight').first()
```
Comment (thorulf4) [inline edufinder/edufinder/rest_api/views.py:73]: Mangler tilfældigheden fra den tidligere løsning.

Comment (thorulf4) [inline edufinder/edufinder/rest_api/views.py:73]: If all elements have the same weight they should probably be picked randomly not in sequence.
Similar to what @joandrsn said.

File: edufinder/edufinder/rest_api/views.py [hunk 2]
```diff
@@ -58,7 +58,8 @@ def get_firstquestion():
     """
     Returns the primary key of the first question asked.
     """
-    return 1
+    return Question.objects.order_by('-weight').first()
```
Comment (thorulf4) [inline edufinder/edufinder/rest_api/views.py:61]: For the data collection the first question probably should also be random.

**Review by thorulf4 — APPROVED** [review]
Except for the 2 comments it seems fine.

**Review by joandrsn — APPROVED** [review]

**Pull request closed without merging**